lib/pf/iplog.pm

```diff
@@ -267,91 +269,105 @@ sub iplog_expire {
     return db_data(IPLOG, $iplog_statements, 'iplog_expire_sql', $time);
 }
 
+=head2 ip2mac
+
+Lookup for the MAC address of a given IP address
+
+Returns '0' if no match
+
+=cut
+
 sub ip2mac {
-    my ( $ip, $date ) = @_;
-    my $logger = Log::Log4perl::get_logger('pf::iplog');
-    my $mac;
+    my ( $ip ) = @_;
+    my $logger = pf::log::get_logger();
+
     return (0) if ( !valid_ip($ip) );
```

Comment:
I know you didn't write that piece, but an error or warn log message would be useful when returning here